schemas/queries/prometheus/query.cue

```diff
@@ -21,6 +21,7 @@ kind: "PrometheusTimeSeriesQuery"
 spec: close({
 	datasource?: {
 		kind: prometheus.kind
+		name?: string
```

Comment:
it looks like the datasource selector missed the `name` of the datasource for the query. Maybe we should create a common `datasource` selector ?

Reply:
Lol this keeps coming back :D the name attribute is already defined in the base definition for queries https://github.com/perses/perses/blob/dbf9cb4c09aea90ab5970bcc7800e1ac786d93e4/internal/api/shared/schemas/base_def_query.cue#L20

This is true that it's kinda misleading / not clear here, but I don't know if we can improve this other than with proper documentation; One might think about using imports here to clearly show that this prometheus query definition is built over a generic one, but doing this would allow the future plugin designers to not do this import thus not comply with the expected datamodel..